railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -16,6 +16,10 @@ Rails.application.configure do
   config.action_controller.perform_caching = true
   <%- end -%>
 
+  # Set canonical url for your application that'll be accessible via Rails.application.url.
+  # Used by Action Controller, Action Mailer, Active Storage for generating urls outside of active requests.
+  # config.application_url = "https://app.example.com"
```

Comment:
Is it reasonable to have this settable with RAILS_APPLICATION_URL in the env as well?  (Maybe in a separate PR).

Reply:
Yeah, that's a good idea to suggest in the comment. `config.application_url = ENV["APP_URL"] || "https://app.example.com"`. And then we can have a comment in `config/deploy.yml` that corresponds to this 👍